backend/isic/crawlinginmyskin.py

```diff
@@ -0,0 +1,137 @@
+import requests
+import pandas as pd
+import pathlib
+import os
+
+"""A module to fetch data from International Skin Imaging Collaboration's API
+For further information about the API: https://isic-archive.com/api/v1/ """
+
+
+def ISIC_request(response, num=100):
+    """Fetching the metadata from images in ISIC archive
+    by sending a HTTP request to the ISIC's REST API
+
+    Parameters:
+
+    num(int):Number of the images you get from the archive,default is 100
+
+    Returns:
+
+    Json:Response metadata in JSON format"""
+    start_url = 'https://isic-archive.com/api/v1/image?limit=' + \
+        str(num)+'&sort=name&sortdir=-1&detail=true'
+    headers = {
+        'Accept': 'application/json'
+    }
+    r = requests.get(start_url, headers=headers)
+    data = r.json()
+    return data
+
+
+def check_dermoscopic(meta):
+    """Checking if a image is acquired through dermoscopy by ISIC's API
+
+    Parameter:
+    meta:The metadata of the image getting through the API
+
+    Return:
+    True if the image is acquired through dermoscopy,False if it isn't
+    """
+
+    if "image_type" not in meta["meta"]["acquisition"]:
+        return False
+    elif not str(meta["meta"]["acquisition"]["image_type"]) == "dermoscopic":
+        return False
+    return True
+
+
+def check_melanoma(metadata):
+    """Checking if a image is confirmed melanocytic by ISIC's API
+
+    Parameter:
+    metadata:The metadata of the image getting through the API
+
+    Return:
+    True if the image is confirmed melanocytic,False if it isn't
+    """
+    if "melanocytic" not in metadata["meta"]["clinical"]:
+        return False
+    elif not metadata["meta"]["clinical"]["melanocytic"]:
+        return False
+    return True
+
+
+def ISIC_getdata(numjson=200, numimage=20,
```

Comment:
This functions should provide returning as a file and an option to export to csv file.

Reply:
"Returning" should be literal data and "export" is for extra files.

Reply:
Yes. That's what I meant. I edited my comments.